pages/thc/[processRef]/confirmed.tsx

```diff
@@ -17,6 +17,8 @@ const ConfirmedPage: NextPage = () => {
   const { status } = router.query;
   const isManagerPage = isManager(router);
 
+  const managerAppraisal = status && status === "2" ? "approved" : "declined";
```

Comment:
If `status` is undefined, we probably want to show `Loading...` on the page instead of all of the text.